src/serialize/value.rs

```diff
@@ -0,0 +1,1075 @@
+use chrono_04::NaiveDate;
+use pyo3::prelude::*;
+use pyo3::types::{PyBytes, PyDict, PyList, PyMapping, PySet, PyTuple};
+use pyo3::{Bound, PyAny, PyErr, PyResult};
+use scylla::cluster::metadata::{CollectionType, ColumnType, NativeType, UserDefinedType};
+use scylla::serialize::SerializationError;
+use scylla::serialize::value::{
+    BuiltinSerializationError, BuiltinSerializationErrorKind, BuiltinTypeCheckError,
+    BuiltinTypeCheckErrorKind, MapSerializationErrorKind, MapTypeCheckErrorKind,
+    SetOrListSerializationErrorKind, SetOrListTypeCheckErrorKind,
+};
+use scylla::serialize::value::{SerializeValue, UdtTypeCheckErrorKind};
+use scylla::serialize::writers::{CellWriter, WrittenCellProof};
+use scylla::value::{
+    Counter, CqlDate, CqlDecimal, CqlDuration, CqlTime, CqlTimestamp, CqlTimeuuid, CqlValue,
+    CqlVarint,
+};
+use scylla_cql::serialize::value::{
+    VectorSerializationErrorKind, serialize_next_variable_length_elem,
+};
+use scylla_cql::serialize::writers::CellValueBuilder;
+use scylla_cql::value::{CqlVarintBorrowed, ValueOverflow};
+use std::cmp::max;
+use std::net::IpAddr;
+use std::ops::Deref;
+use std::sync::Arc;
+use thiserror::Error;
+use time_03::{OffsetDateTime, Time};
```

Comment:
:wrench: I can see you're using `time` types. Can we use `chrono` types instead? I believe they are preferred.